docs/install-upgrade/build-wiring.md

```diff
@@ -277,5 +256,5 @@ The dotted line in the diagram shows the traffic flow for remote peering. The tr
 
 #### VPC Loopback
 
-A VPC loopback is a physical cable with both ends plugged into the same switch, suggested but not required to be the adjacent ports. This loopback allows two different VPCs to communicate with each other. This is due to a Broadcom limitation.
+A VPC loopback consists of a physical cable with both ends connected to the same switch, typically using adjacent ports. This configuration enables communication between different VPCs, addressing a Broadcom limitation.
```

Comment:
My understanding is that this is required for communication. Can we add the requirement into here?